JCL_Test_0 failed in VT standard build

https://openj9-jenkins.osuosl.org/job/Test_openjdknext_j9_sanity.functional_aarch64_linux_vt_standard_Personal_testList_0/15/

```
15:12:30  FAILED: test_getModifiers_classTypes
15:12:30  java.lang.AssertionError: One or more tests failed:
15:12:30  Incorrect modifiers for SyntheticClass. Expected <4129>, Actual <4097>.
15:12:30  
15:12:30  	at org.testng.Assert.fail(Assert.java:96) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.openj9.test.java.lang.Test_Class.test_getModifiers_classTypes(Test_Class.java:1043) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/workspace/Test_openjdknext_j9_sanity.functional_aarch64_linux_vt_standard_Personal_testList_0/jvmtest/functional/Java8andUp/GeneralTest.jar)
15:12:30  	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) from jrt:/java.base
15:12:30  	at java.base/java.lang.reflect.Method.invoke(Method.java:573) from jrt:/java.base
15:12:30  	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestRunner.privateRun(TestRunner.java:648) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestRunner.run(TestRunner.java:505) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunner.run(SuiteRunner.java:364) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.runSuites(TestNG.java:1049) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.run(TestNG.java:1017) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.privateMain(TestNG.java:1354) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
15:12:30  	at org.testng.TestNG.main(TestNG.java:1323) from jdk.internal.loader.ClassLoaders$AppClassLoader@12441d7a(file:/home/jenkins/testDependency/lib/testng.jar)
```

--- a/test/functional/Java8andUp/src_110_up/org/openj9/test/java/lang/Test_Class.java
+++ b/test/functional/Java8andUp/src_110_up/org/openj9/test/java/lang/Test_Class.java
@@ -1023,7 +1023,7 @@ public void test_getModifiers_classTypes() {
 		0x0000 | IDENTITY, // (Identity)
 		0x0410,            // static
 		0x0400 | IDENTITY, // abstract (Identity)
-		0x1001 | IDENTITY, // public synthetic (Identity)
+		0x1001,            // public synthetic
 		0x2608,            // static abstract interface annotation
 		0x4018 | IDENTITY, // static final enum (Identity)
 	};

